custom/icds_reports/reports/disha.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, state_name, month):
         self.month = month
 
     def _blob_id(self):
-        return 'disha_dump-{}-{}.json'.format(self.state_name.replace(" ", ""), self.month.strftime('%Y-%m-%d'))
+        return 'disha_dump-{}-{}.json'.format(self.state_name.encode('ascii', 'ignore').replace(" ", ""), self.month.strftime('%Y-%m-%d'))
```

Comment:
What was the state name that caused this? `Madhya Pradesh`?

Reply:
There was no issue here, I just took a precaution. This would break if the state-name as unicode only chars.